source/Tree/Tree.example.js

```diff
@@ -0,0 +1,238 @@
+import React from 'react';
+import {AutoSizer} from '../AutoSizer';
+import {
+  ContentBox,
+  ContentBoxHeader,
+  ContentBoxParagraph,
+} from '../demo/ContentBox';
+import Tree from './Tree';
+import {InputRow, LabeledInput} from '../demo/LabeledInput';
+import styles from './Tree.example.css';
+
+function getRandomInt(min, max) {
+  return Math.floor(Math.random() * (max - min)) + min;
+}
+
+export default class TreeExample extends React.PureComponent {
+  static overscanIndicesGetter({startIndex, stopIndex}) {
```

Comment:
I'm curious- is this necessary?

This is kind of an advanced property that I don't expect to be used very often, so I'm surprised to see it in the demo. It should almost always be sufficient to specify `overscanRowCount`.

Just curious if this hints at some performance issue or limitation?

Reply:
Hmm... That's strange. I have no idea why this code is here. I think it would be better if I remove it.